pir/cpp/context.h

```diff
@@ -47,27 +49,28 @@ class PIRContext {
   /**
    * Returns the PIR parameters
```

Comment:
Perhaps give a little more context as to what these are? Also that it shouldn't be used for encryption parameters.